cpp/src/arrow/testing/random.h

```diff
@@ -322,6 +322,16 @@ class ARROW_TESTING_EXPORT RandomArrayGenerator {
   std::shared_ptr<Array> FixedSizeBinary(int64_t size, int32_t byte_width,
                                          double null_probability = 0);
 
+  // /// \brief Generate a random StructArray
```

Comment:
Why the doubled comment+docstring prefix?

Reply:
If this is not implemented, please leave it out :-)

Reply:
Fixed. This looks like a relic from an old commit. Sorry.